hs/src/Reach/AST/DLBase.hs

```diff
@@ -53,7 +54,7 @@ data DLType
   | T_Token
   | T_Array DLType Integer
   | T_Tuple [DLType]
-  | T_Object (M.Map SLVar DLType)
+  | T_Object (M.Map SLVar (SecurityLevel, DLType))
```

Comment:
No no no, we can't do that. Security levels cannot exist at run-time

Reply:
okay, I'll try to work it out another way